2015-11-16T10:11:30.083370+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/arbiter.py", line 459, in reap_workers
2015-11-16T10:11:30.083606+00:00 app[web.1]:     raise HaltServer(reason, self.WORKER_BOOT_ERROR)
2015-11-16T10:11:30.083089+00:00 app[web.1]:     self.manage_workers()
2015-11-16T10:11:30.083237+00:00 app[web.1]:     self.spawn_workers()
2015-11-16T10:11:30.083608+00:00 app[web.1]: gunicorn.errors.HaltServer: <HaltServer 'Worker failed to boot.' 3>
2015-11-16T10:11:30.083270+00:00 app[web.1]:     time.sleep(0.1 * random.random())
2015-11-16T10:11:30.083291+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/arbiter.py", line 214, in handle_chld
2015-11-16T10:11:30.954709+00:00 heroku[web.1]: Process exited with status 1
2015-11-16T10:11:30.966201+00:00 heroku[web.1]: State changed from starting to crashed
2015-11-16T10:11:46.398243+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=cse-rrn6237-2.herokuapp.com request_id=21e68c8d-8c07-40b3-a25f-464ec47f9926 fwd="76.185.80.245" dyno= connect= service= status=503 bytes=
2015-11-16T10:11:46.965710+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=cse-rrn6237-2.herokuapp.com request_id=a4f61fc8-8beb-4101-a6f2-0639615e9b6f fwd="76.185.80.245" dyno= connect= service= status=503 bytes=
2015-11-16T10:15:29.660729+00:00 heroku[web.1]: State changed from crashed to starting
2015-11-16T10:15:33.291813+00:00 heroku[web.1]: Starting process with command `gunicorn app:app`
2015-11-16T10:15:35.861524+00:00 app[web.1]: [2015-11-16 10:15:35 +0000] [3] [INFO] Starting gunicorn 19.3.0
2015-11-16T10:15:35.870775+00:00 app[web.1]: [2015-11-16 10:15:35 +0000] [9] [ERROR] Exception in worker process:
2015-11-16T10:15:35.870786+00:00 app[web.1]:     return self.load_wsgiapp()
2015-11-16T10:15:35.870791+00:00 app[web.1]: Traceback (most recent call last):
2015-11-16T10:15:35.870779+00:00 app[web.1]: Traceback (most recent call last):
2015-11-16T10:15:35.870789+00:00 app[web.1]:     __import__(module)
2015-11-16T10:15:35.870781+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/arbiter.py", line 507, in spawn_worker
2015-11-16T10:15:35.870788+00:00 app[web.1]:     return util.import_app(self.app_uri)
2015-11-16T10:15:35.870782+00:00 app[web.1]:     worker.init_process()
2015-11-16T10:15:35.870785+00:00 app[web.1]:     self.callable = self.load()
2015-11-16T10:15:35.870783+00:00 app[web.1]:     self.wsgi = self.app.wsgi()
2015-11-16T10:15:35.870791+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/arbiter.py", line 507, in spawn_worker
2015-11-16T10:15:35.870783+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/workers/base.py", line 118, in init_process
2015-11-16T10:15:35.870784+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
2015-11-16T10:15:35.870790+00:00 app[web.1]: ImportError: No module named app
2015-11-16T10:15:35.870786+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
2015-11-16T10:15:35.870787+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
2015-11-16T10:15:35.870788+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/util.py", line 355, in import_app
2015-11-16T10:15:35.870792+00:00 app[web.1]:     worker.init_process()
2015-11-16T10:15:35.870793+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/workers/base.py", line 118, in init_process
2015-11-16T10:15:35.870798+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/util.py", line 355, in import_app
2015-11-16T10:15:35.891329+00:00 app[web.1]:   File "/app/.heroku/python/bin/gunicorn", line 11, in <module>
2015-11-16T10:15:35.870793+00:00 app[web.1]:     self.wsgi = self.app.wsgi()
2015-11-16T10:15:35.870797+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
2015-11-16T10:15:35.891500+00:00 app[web.1]:     WSGIApplication("%(prog)s [OPTIONS] [APP_MODULE]").run()
2015-11-16T10:15:35.891597+00:00 app[web.1]:     super(Application, self).run()
2015-11-16T10:15:35.870794+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
2015-11-16T10:15:35.870796+00:00 app[web.1]:     return self.load_wsgiapp()
2015-11-16T10:15:35.870794+00:00 app[web.1]:     self.callable = self.load()
2015-11-16T10:15:35.891417+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 74, in run
2015-11-16T10:15:35.891636+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/app/base.py", line 72, in run
2015-11-16T10:15:35.870795+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
2015-11-16T10:15:35.891529+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/app/base.py", line 189, in run
2015-11-16T10:15:35.891726+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/arbiter.py", line 174, in run
2015-11-16T10:15:35.870797+00:00 app[web.1]:     return util.import_app(self.app_uri)
2015-11-16T10:15:35.891280+00:00 app[web.1]: Traceback (most recent call last):
2015-11-16T10:15:35.891810+00:00 app[web.1]:     self.manage_workers()
2015-11-16T10:15:35.870799+00:00 app[web.1]:     __import__(module)
2015-11-16T10:15:35.891388+00:00 app[web.1]:     sys.exit(run())
2015-11-16T10:15:35.870799+00:00 app[web.1]: ImportError: No module named app
2015-11-16T10:15:35.870952+00:00 app[web.1]: [2015-11-16 10:15:35 +0000] [9] [INFO] Worker exiting (pid: 9)
2015-11-16T10:15:35.891701+00:00 app[web.1]:     Arbiter(self).run()
2015-11-16T10:15:35.862199+00:00 app[web.1]: [2015-11-16 10:15:35 +0000] [3] [INFO] Listening at: http://0.0.0.0:45032 (3)
2015-11-16T10:15:35.862344+00:00 app[web.1]: [2015-11-16 10:15:35 +0000] [3] [INFO] Using worker: sync
2015-11-16T10:15:35.867978+00:00 app[web.1]: [2015-11-16 10:15:35 +0000] [9] [INFO] Booting worker with pid: 9
2015-11-16T10:15:35.891849+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/arbiter.py", line 477, in manage_workers
2015-11-16T10:15:35.892204+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/arbiter.py", line 214, in handle_chld
2015-11-16T10:15:35.891960+00:00 app[web.1]:     self.spawn_workers()
2015-11-16T10:15:35.892008+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/arbiter.py", line 541, in spawn_workers
2015-11-16T10:15:35.892179+00:00 app[web.1]:     time.sleep(0.1 * random.random())
2015-11-16T10:15:35.892270+00:00 app[web.1]:     self.reap_workers()
2015-11-16T10:15:35.892405+00:00 app[web.1]:     raise HaltServer(reason, self.WORKER_BOOT_ERROR)
2015-11-16T10:15:35.892291+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/arbiter.py", line 459, in reap_workers
2015-11-16T10:15:35.892531+00:00 app[web.1]: gunicorn.errors.HaltServer: <HaltServer 'Worker failed to boot.' 3>
2015-11-16T10:15:36.993038+00:00 heroku[web.1]: Process exited with status 1
2015-11-16T10:15:37.013006+00:00 heroku[web.1]: State changed from starting to crashed
2015-11-16T10:17:07.136615+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=cse-rrn6237-2.herokuapp.com request_id=7a254f35-952d-4bbc-8c83-f57031d7fe3e fwd="76.185.80.245" dyno= connect= service= status=503 bytes=
2015-11-16T10:17:07.519156+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=cse-rrn6237-2.herokuapp.com request_id=d650de0e-7aba-4dd3-aacc-fa17fd728f9f fwd="76.185.80.245" dyno= connect= service= status=503 bytes=
2015-11-16T10:25:50.065450+00:00 heroku[api]: Deploy 03322bf by rajesh.nayak@mavs.uta.edu
2015-11-16T10:25:50.065450+00:00 heroku[api]: Release v7 created by rajesh.nayak@mavs.uta.edu
2015-11-16T10:25:50.234704+00:00 heroku[slug-compiler]: Slug compilation started
2015-11-16T10:25:50.234722+00:00 heroku[slug-compiler]: Slug compilation finished
2015-11-16T10:25:50.250760+00:00 heroku[web.1]: State changed from crashed to starting
2015-11-16T10:25:53.058727+00:00 heroku[web.1]: Starting process with command `gunicorn app:app`
2015-11-16T10:25:55.204358+00:00 app[web.1]: [2015-11-16 10:25:55 +0000] [3] [INFO] Starting gunicorn 19.3.0
2015-11-16T10:25:55.304449+00:00 app[web.1]: [2015-11-16 10:25:55 +0000] [10] [INFO] Booting worker with pid: 10
2015-11-16T10:25:55.204806+00:00 app[web.1]: [2015-11-16 10:25:55 +0000] [3] [INFO] Listening at: http://0.0.0.0:13803 (3)
2015-11-16T10:25:55.204841+00:00 app[web.1]: [2015-11-16 10:25:55 +0000] [3] [INFO] Using worker: sync
2015-11-16T10:25:55.208486+00:00 app[web.1]: [2015-11-16 10:25:55 +0000] [9] [INFO] Booting worker with pid: 9
2015-11-16T10:25:55.981272+00:00 app[web.1]: /app/.heroku/python/lib/python2.7/site-packages/flask_sqlalchemy/__init__.py:800: UserWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True to suppress this warning.
2015-11-16T10:25:55.981276+00:00 app[web.1]:   warnings.warn('SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True to suppress this warning.')
2015-11-16T10:25:55.983850+00:00 app[web.1]: /app/.heroku/python/lib/python2.7/site-packages/flask_sqlalchemy/__init__.py:800: UserWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True to suppress this warning.
2015-11-16T10:25:55.983852+00:00 app[web.1]:   warnings.warn('SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True to suppress this warning.')
2015-11-16T10:25:56.726456+00:00 heroku[web.1]: State changed from starting to up
2015-11-16T10:26:20.517180+00:00 heroku[router]: at=info method=GET path="/" host=cse-rrn6237-2.herokuapp.com request_id=c7a4cf8c-e774-442b-8e47-2549071d79fe fwd="76.185.80.245" dyno=web.1 connect=1ms service=2ms status=200 bytes=187
2015-11-16T10:26:28.032716+00:00 heroku[router]: at=info method=GET path="/favicon.ico" host=cse-rrn6237-2.herokuapp.com request_id=1b3d66bb-553a-4b6f-b352-c95eaedf3e38 fwd="76.185.80.245" dyno=web.1 connect=1ms service=114ms status=404 bytes=391
2015-11-16T10:26:44.126533+00:00 heroku[router]: at=info method=GET path="/lotsofdata" host=cse-rrn6237-2.herokuapp.com request_id=fd6d17ca-e7a8-4e9d-aab5-af2a53bd919e fwd="76.185.80.245" dyno=web.1 connect=0ms service=136ms status=500 bytes=456
2015-11-16T10:26:59.844645+00:00 heroku[router]: at=info method=GET path="/lotsofdata" host=cse-rrn6237-2.herokuapp.com request_id=0452e65f-7459-4bca-8126-8481df89567e fwd="76.185.80.245" dyno=web.1 connect=1ms service=110ms status=500 bytes=456
2015-11-16T10:27:01.653772+00:00 heroku[router]: at=info method=GET path="/lotsofdata" host=cse-rrn6237-2.herokuapp.com request_id=3e9e6e6f-9ca0-4cd7-a09c-eae2910682ef fwd="76.185.80.245" dyno=web.1 connect=1ms service=130ms status=500 bytes=456
2015-11-16T10:27:03.145173+00:00 heroku[router]: at=info method=GET path="/lotsofdata" host=cse-rrn6237-2.herokuapp.com request_id=23f9a986-c1b1-4fef-8bf7-51d4d49fd2fa fwd="76.185.80.245" dyno=web.1 connect=1ms service=106ms status=500 bytes=456
2015-11-16T10:27:04.308874+00:00 heroku[router]: at=info method=GET path="/lotsofdata" host=cse-rrn6237-2.herokuapp.com request_id=ef4f2195-bf8e-4334-82cc-f6026965f2f2 fwd="76.185.80.245" dyno=web.1 connect=2ms service=60ms status=500 bytes=456
2015-11-16T10:27:53.100150+00:00 heroku[router]: at=info method=GET path="/lotsofdata" host=cse-rrn6237-2.herokuapp.com request_id=192dcb1b-6ceb-4e18-97b0-e9cdf3d06645 fwd="76.185.80.245" dyno=web.1 connect=0ms service=82ms status=500 bytes=456
2015-11-16T10:28:25.455097+00:00 heroku[router]: at=info method=GET path="/lotsofdata" host=cse-rrn6237-2.herokuapp.com request_id=d756115c-01f0-4dc3-a469-3df79d9d09fc fwd="76.185.80.245" dyno=web.1 connect=1ms service=88ms status=500 bytes=456
2015-11-16T10:28:28.461341+00:00 heroku[router]: at=info method=GET path="/lotsofdata" host=cse-rrn6237-2.herokuapp.com request_id=319b6887-3c2a-432e-bcdf-9af5076270f7 fwd="76.185.80.245" dyno=web.1 connect=1ms service=54ms status=500 bytes=456
2015-11-16T10:35:19.069601+00:00 heroku[router]: at=info method=GET path="/lotsofdata" host=cse-rrn6237-2.herokuapp.com request_id=a9c71726-3b91-4aad-a57d-1a56ca3a4b58 fwd="76.185.80.245" dyno=web.1 connect=0ms service=207ms status=500 bytes=456
2015-11-16T10:35:20.506210+00:00 heroku[router]: at=info method=GET path="/lotsofdata" host=cse-rrn6237-2.herokuapp.com request_id=e414065f-f8a8-482f-a202-23a7ad39912a fwd="76.185.80.245" dyno=web.1 connect=0ms service=58ms status=500 bytes=456
2015-11-16T10:35:21.759651+00:00 heroku[router]: at=info method=GET path="/lotsofdata" host=cse-rrn6237-2.herokuapp.com request_id=cfca4f9f-c100-4487-939a-8b803f207316 fwd="76.185.80.245" dyno=web.1 connect=11ms service=54ms status=500 bytes=456
2015-11-16T10:35:22.701392+00:00 heroku[router]: at=info method=GET path="/lotsofdata" host=cse-rrn6237-2.herokuapp.com request_id=ae2271be-53fe-4552-aa95-e743b8ac6918 fwd="76.185.80.245" dyno=web.1 connect=1ms service=97ms status=500 bytes=456
